src/gowrapper/ckks/evaluator.go

```diff
@@ -134,6 +137,10 @@ func lattigo_rescaleMany(evalHandle Handle4, paramsHandle Handle4, ctInHandle Ha
 		targetScale /= (float64(params.RingQ().Modulus[ctIn.Level()-i]))
 	}
 
+	if targetScale <= 0 {
+		panic(errors.New("Target scale is too small: " + strconv.FormatFloat(targetScale, 'E', -1, 64) + "\t" + strconv.FormatFloat(ctIn.Scale, 'E', -1, 64) + "\t" + strconv.FormatFloat(math.Log2(ctIn.Scale), 'E', -1, 64) + "\t" + strconv.FormatUint(numRescales, 10)))
+	}
```

Comment:
`panic` exits program, right? this line may just return this error?
*  `panic` is used when the program, or its part, has reached an unrecoverable state.

Reply:
This wrapper does not return errors to the user, it always just exits.

Reply:
Discussed in slack. Not a blocker.